data/governments.txt

```diff
@@ -454,6 +454,29 @@ government "Hai (Wormhole Access)"
 	"hostile hail" "hostile hai"
 	"hostile disabled hail" "hostile disabled hai"
 
+# A government that is used to allow Unfettered raids in Wah Ki and nearby systems.
+government "Hai (Raided)"
+	swizzle 0
+	color "governments: Hai"
+	
+	"player reputation" 0
+	"attitude toward"
+		"Hai (Wormhole Access)" 1
+		"Hai Merchant" 1
+		"Hai Merchant (Sympathizers)" 1
+		"Hai Merchant (Human)" 1
+		"Hai (Unfettered)" -.1
+		"Hai (Friendly Unfettered)" .1
+		"Hai (Unfettered Civilians)" .1
+		"Merchant" .01
+		"Elenctic Commune" .1
+	"bribe" .2
```

Comment:
```suggestion
	"bribe" .02
```

Just noticed this, too.

To match the others. Or is this deliberately different and not a typo?

Reply:
look above, the main hai gov has .2